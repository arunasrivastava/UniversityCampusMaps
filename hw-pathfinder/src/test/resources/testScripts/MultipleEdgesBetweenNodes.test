# test dijkstra to choose edges over nodes

CreateGraph g
AddNode g c1
AddNode g c2
AddNode g c3
AddNode g c4
AddNode g c5
AddEdge g c1 c2 1.1
AddEdge g c1 c3 1.05
AddEdge g c1 c4 1.05
AddEdge g c2 c5 1
AddEdge g c3 c5 1
AddEdge g c4 c5 1

FindPath g c1 c5

# test dijkstra to choose edges over nodes

CreateGraph g2
AddNode g2 c1
AddNode g2 c2
AddNode g2 c5
AddEdge g2 c1 c2 1.1
AddEdge g2 c1 c2 1.05
AddEdge g2 c1 c2 1
AddEdge g2 c1 c2 1.001
AddEdge g2 c2 c5 1.09
AddEdge g2 c2 c5 1.08

FindPath g2 c1 c5