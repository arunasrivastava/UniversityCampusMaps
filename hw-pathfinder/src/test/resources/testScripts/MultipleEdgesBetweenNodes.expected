# test dijkstra to choose edges over nodes

created graph g
added node c1 to g
added node c2 to g
added node c3 to g
added node c4 to g
added node c5 to g
added edge 1.100 from c1 to c2 in g
added edge 1.050 from c1 to c3 in g
added edge 1.050 from c1 to c4 in g
added edge 1.000 from c2 to c5 in g
added edge 1.000 from c3 to c5 in g
added edge 1.000 from c4 to c5 in g

path from c1 to c5:
c1 to c4 with weight 1.050
c4 to c5 with weight 1.000
total cost: 2.050

# test dijkstra to choose edges over nodes

created graph g2
added node c1 to g2
added node c2 to g2
added node c5 to g2
added edge 1.100 from c1 to c2 in g2
added edge 1.050 from c1 to c2 in g2
added edge 1.000 from c1 to c2 in g2
added edge 1.001 from c1 to c2 in g2
added edge 1.090 from c2 to c5 in g2
added edge 1.080 from c2 to c5 in g2

path from c1 to c5:
c1 to c2 with weight 1.000
c2 to c5 with weight 1.080
total cost: 2.080